torchgeo/trainers/cyclone.py

```diff
@@ -36,7 +36,7 @@ def config_task(self, kwargs: Dict[str, Any]) -> None:
         else:
             raise ValueError(f"Model type '{kwargs['model']}' is not valid.")
 
-    def __init__(self, **kwargs: Dict[str, Any]) -> None:
+    def __init__(self, **kwargs: Any) -> None:
```

Comment:
Why the switch to `Any` here? In mypy language, `Any` basically means "skip type checks", so I'm trying to avoid it whenever possible. In the future, I would like to never use `Any` anywhere in the code base. For dicts like `**kwargs` I see no reason not to specify that it has to be a dict.

Reply:
Dict isn't needed when annotating **kwargs, see https://adamj.eu/tech/2021/05/11/python-type-hints-args-and-kwargs/ or https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values. As kwargs here can actually be Any I think it is legit

Reply:
Oh cool! I guess ya learn something new every day